### PR TITLE
CHANGELOG: add release notes for v2.1.8

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

release notes for v2.1.8

### CODE REVIEW

**Comment by winkyao** [comment]
@jackysp @zz-jason PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.8] 2019-04-12
+
+* Fix the issue that the processing logic of `GROUP_CONCAT` function is incompatible with MySQL when there is a NULL-valued parameter [#9930](https://github.com/pingcap/tidb/pull/9930)
+* Fix the equality check issue of decimal values in the `Distinct` mode [#9931](https://github.com/pingcap/tidb/pull/9931)
+* Fix the collation compatibility issue of the date, datetime, and timestamp types for the `SHOW FULL COLUMNS` statement 
+    - [#9938](https://github.com/pingcap/tidb/pull/9938)
+    - [#10114](https://github.com/pingcap/tidb/pull/10114)
+* Support committing transactions in batches for DML statements, enabled using `enable-batch-dml` [#9882](https://github.com/pingcap/tidb/pull/9882)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+* Fix the issue that the row count estimation is inaccurate when the filtering condition contains correlated columns [#9937](https://github.com/pingcap/tidb/pull/9937)
+* Fix the compatibility issue between the `DATE_ADD` and `DATE_SUB` functions
+    - [#9963](https://github.com/pingcap/tidb/pull/9963)
+    - [#9966](https://github.com/pingcap/tidb/pull/9966)
+* Support the `%H` format for the `STR_TO_DATE` function to improve compatibility [#9964](https://github.com/pingcap/tidb/pull/9964)
+* Fix the issue that the result is wrong when the `GROUP_CONCAT` function groups by a unique index [#9969](https://github.com/pingcap/tidb/pull/9969)
+* Add the `txn-entry-count-limit` configuration item to modify the maximum number of statements of a transaction and the `txn-total-size-limit` configuration item to modify the maximum size in bytes of a transaction [#9952](https://github.com/pingcap/tidb/pull/9952)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
```

**Comment by winkyao** [comment]
@jackysp PTAL

**Pull request merged**